navigation/index.tsx

```diff
@@ -9,6 +9,7 @@ import Login from "@/screens/Authentication/Login"
 import CustomSplashScreen from "@/screens/CustomSplashScreen"
 import Settings from "@/screens/Settings"
 import Signaling from "@/components/lists/Signaling"
+import EditAvailability from "@/screens/EditActivity"
```

Comment:
let's import this as import `EditAvailability`
```suggestion
import EditActivity from "@/screens/EditActivity"
```